dotnet/Microsoft.MacCatalyst.Sdk/targets/Microsoft.MacCatalyst.Sdk.targets

```diff
@@ -4,6 +4,6 @@
 
   <PropertyGroup>
     <RunCommand>open</RunCommand>
-    <RunArguments>$(OutputPath)/$(AssemblyName).app</RunArguments>
+    <RunArguments>$([MSBuild]::NormalizePath($(OutputPath)/$(AssemblyName).app))</RunArguments>
```

Comment:
I realized that the output directory of the assembly was not resolved when I ran the `dotnet run` command with the `--project` option from a directory other than the one where the project file exists.

For example, set cwd to MyProject.sln dir, and run `dotnet run --project MyProject`, then `RunArguments` become    `$(MyProject.sln dir)/bin/Debug/MyProject.app`
```
MyProject.sln
|- MyProject
|- MyProjectLib
```



I think a good way to deal with this is to replace `OutputPath` with `TargetDir` , what do you think?

Reply:
This is the real problem: if you just replace `OutputPath` with `TargetDir` (without calling `NormalizePath`), then it works.

Reply:
Could you make this change? Just change `OutputPath`  to `TargetDir` (and remove the call to `NormalizePath`, it should be done automatically by MSBuild if the path exists).

Reply:
Thank you for your suggestions. 
I fixed it https://github.com/xamarin/xamarin-macios/pull/11994/commits/723aff85c0db20af976c1f4d50a61cb8ecd06e84 .